.github/workflows/pkg.pr.new.yml

```
@@ -12,6 +12,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332
+      - name: Check for changes to nuqs package
+        id: check-for-changes
+        run: |
+          if git diff --quiet HEAD^ HEAD ./packages/nuqs; then
```

Now for the Git diff part, we could try this (though `github.event.pull_request.base.sha` might not be right in case we rebase the PR, TBC).

```suggestion
          if git diff --quiet ${{ github.event.pull_request.base.sha }} ${{ github.event.pull_request.head.sha }} ./packages/nuqs; then
```